.github/workflows/console-preview.yml

```diff
@@ -0,0 +1,68 @@
+name: Console Preview
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - closed
+    paths-ignore:
+      - "docs/**"
+      - "logo/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.head.ref || github.ref }}
+  cancel-in-progress: true
+
+permissions: 
+   pull-requests: write
+
+env:
+  FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+  APP_NAME: wing-console-pr-${{github.event.number}}
+
+jobs:
+  create:
+    name: "Create"
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.head.repo.full_name == github.repository && github.event.action != 'closed' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
```

Comment:
Maybe we should pin to a specific version, and update manually from time to time?